pylanetary/navigation/core.py

```diff
@@ -148,33 +148,43 @@ def emission_angle(ob_lat, surf_n):
     
     Parameters
     ----------
+    ob_lat : float or np.array, required. 
+        [deg] sub-observer latitude
+    surf_n : np.array, required.
+        [x,y,z] surface normal vector at each ob_lat. must have vector length 1
```

Comment:
This is personal preference, but I would normalize the surface vector instead of assuming it's normalized